runtime/src/lib.rs

```diff
@@ -477,6 +480,34 @@ impl pallet_sudo::Config for Runtime {
 	type WeightInfo = ();
 }
 
+pub struct AccountMapping;
+impl pallet_living_assets_ownership::traits::AccountMapping<AccountId> for AccountMapping {
```

Comment:
same here, it is more readable if you import the traits in the beginning of the file. This looks long and repetitive

Reply:
also some comment on how conversion is done would be nice

Reply:
and also, do you think we can use `Convert` trait instead of defining new one?

```rs
impl Convert<AccountId, H160> for AccountMapping<AccountId, H160> {
   fn convert(from: AccountId) -> H160 {}
}
```